### DISCUSSION TITLE
Send email to all registered customer?

### DISCUSSION THREAD

**Discussion by stblassitude** [pretix/pretix#4535] [Q&A] [answered]
Is there a way to send email on the organizer level to all registered customers? (This is on pretix.eu) Or are customers only associated with our organizer once they have bought something?

**Comment by pc-coholic** [comment, marked as answer]
There is no way to send an email from the organizer level.

Within an event, where people have purchased something (or are on the waitlist), they can be sent an email using the "send email"-plugin.

**Reply by stblassitude** [reply]
Would you consider a feature request to enable this?
